options.go

```diff
@@ -17,80 +17,74 @@
 package edgedb
 
 import (
-	"fmt"
-	"net/url"
-	"strconv"
-	"strings"
+	"time"
 )
 
 // Options for connecting to an EdgeDB server
 type Options struct {
```

Comment:
Eventually, we want to hide these as private fields and use builder/fluent interface. I.e. only expose methods I think. Not sure if this has to be the same PR.